.github/workflows/scripts/release-against-rancher.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Bumps Webhook version in a locally checked out rancher/rancher repository
+#
+# Usage:
+#   ./release-against-rancher.sh <path to rancher repo> <new webhook release>
+#
+# Example:
+# ./release-against-charts.sh "${GITHUB_WORKSPACE}" "0.5.0-rc.14"
+
+RANCHER_DIR=$1
+NEW_WEBHOOK_VERSION=$2   # e.g. 0.5.2-rc.3
+
+usage() {
+    cat <<EOF
+Usage:
+  $0 <path to rancher repo> <new webhook release>
+EOF
+}
+
+if [ -z "$RANCHER_DIR" ] || [ -z "$NEW_WEBHOOK_VERSION" ]; then
+    usage
+    exit 1
+fi
+
+set -ue
+
+pushd "${RANCHER_DIR}" > /dev/null
+
+# Validate the given webhook version (eg: 0.5.0-rc.13)
+if grep -q "${NEW_WEBHOOK_VERSION}" ./build.yaml; then
+    echo "build.yaml already at version v${NEW_WEBHOOK_VERSION}"
+    exit 1
+fi
+
+# Get the chart version (eg: 104.0.0)
+if ! PREV_CHART_VERSION=$(yq -r '.webhookVersion' ./build.yaml | cut -d+ -f1); then
+    echo "Unable to get chart version from ./build.yaml. The content of the file is:"
+    cat ./build.yaml
+    exit 1
+fi
+NEW_CHART_VERSION=$PREV_CHART_VERSION
+
+yq --inplace ".webhookVersion = \"${NEW_CHART_VERSION}+up${NEW_WEBHOOK_VERSION}\"" ./build.yaml
+
+# Downloads dapper
+make .dapper
+
+# DAPPER_MODE=bind will make sure we output everything that changed
+DAPPER_MODE=bind ./.dapper go generate ./... || true
```

Comment:
Do we need to `go generate` everything here or intent was only to call `go run ./pkg/codegen/buildconfig/...`?

Reply:
I could extract the commands that `go generate` runs but I think it's more future proof to use `go generate` directly. The tradeoff is that this takes longer to run (which also means preventing other jobs to run).

Reply:
It also has potential of failing doing the irrelevant work.

Reply:
I'll leave as is for now.